meeteval/wer/wer/time_constrained.py

```diff
@@ -370,12 +381,53 @@ def sort_segments(s: TimeMarkedTranscript):
     )
 
 
+def sort_and_validate( segments, sort, pseudo_word_level_timing, name):
+    # Check that all timings are valid
+    if len(segments.transcript) != len(segments.timings):
+        raise ValueError(
+            f'Number of words does not match number of timings in {name}: '
+            f'{len(segments.transcript)} != {len(segments.timings)}'
+        )
+
+    for t in segments.timings:
+        if t[1] < t[0]:
+            raise ValueError(f'The end time of an interval must be larger than the start time. Found {t} in {name}')
+
+    prune = True
+
+    assert sort in (True, False, 'segment', 'word')
+    if sort in (True, 'segment', 'word'):
+        segments = sort_segments(segments)
+
+    words = get_pseudo_word_level_timings(segments, pseudo_word_level_timing)
+
+    # Check whether words are sorted by start time
+    words_sorted = sort_segments(words)
+    if words_sorted != words:
```

Comment:
Should this be in a `elif` after the `if sort == 'word':`?

Reply:
I actually wanted to print this warning in all cases. Even if `sort == 'word'` the warning gives you a hint whether the sorting changes the order or not

Reply:
Yes, you are right.